test/generators/suspenders/jobs_generator_test.rb

```diff
@@ -0,0 +1,122 @@
+require "test_helper"
+require "generators/suspenders/jobs_generator"
+
+module Suspenders
+  module Generators
+    class JobsGeneratorTest < Rails::Generators::TestCase
+      include Suspenders::TestHelpers
+
+      tests Suspenders::Generators::JobsGenerator
+      destination Rails.root
+      setup :prepare_destination
+      teardown :restore_destination
+
+      test "adds gems to Gemfile" do
+        expected_output = <<~RUBY
+          gem "sidekiq"
+        RUBY
+
+        run_generator
+
+        assert_file app_root("Gemfile") do |file|
+          assert_match(expected_output, file)
+        end
+      end
+
+      test "installs gems with Bundler" do
+        output = run_generator
+
+        assert_match(/bundle install/, output)
+      end
+
+      test "generator has a description" do
+        description = "Installs sidekiq for background job processing."
+
+        assert_equal description, Suspenders::Generators::JobsGenerator.desc
+      end
+
+      test "configures ActiveJob logging" do
+        expected_configuration = <<~RUBY
+          require "active_job/logging"
+          require "active_job/log_subscriber"
+
+          ActiveSupport::Notifications.unsubscribe("enqueue.active_job")
+
+          module ActiveJob
+            module Logging
+              class EnqueueLogSubscriber < LogSubscriber
+                define_method :enqueue, instance_method(:enqueue)
+              end
+            end
+          end
+
+          ActiveJob::Logging::EnqueueLogSubscriber.attach_to(:active_job)
+        RUBY
+
+        run_generator
+
+        assert_file app_root("config/initializers/active_job.rb") do |file|
+          assert_match(expected_configuration, file)
```

Comment:
[Minor] Does this work?

```suggestion
          assert_equal(expected_configuration, file)
```

Reply:
Yes, it works 👍 I will update it.